configs/meta_terms.yaml

```diff
@@ -0,0 +1,24 @@
+# Some entities/articles are meta information (redirections, categorizations, disambiguations, ...). These are not
+# useful for our candidate retrieval and only slow things down.
+# Hence we compare entities/articles with a list of terms, as unfortunately there doesn't seem to be a language-agnostic
+# mark-up code indicating meta status (or is there?), so we gather them per language.
+
+# Matched against article text.
+articles:
+  en:
+    - "#redirection"
+    - "#redirect"
+    - "{{disambiguation}}"
+  es:
+    - "#redirect"
+    - "#redirección"
+    - "{{desambiguación}}"
+
+# Matched against entity description.
+entities:
+  en:
+    - "Wikimedia category"
+    - "Wikimedia disambiguation page"
+  es:
+    - "categoría de Wikimedia"
+    - "página de desambiguación de Wikimedia"
```

Comment:
I like how you've recorded these. Just FYI, I also had a slightly longer list of "meta" type pages here: https://github.com/explosion/projects/blob/master/nel-wikipedia/wiki_namespaces.py (but not sorted out by language as nicely as here)

Reply:
Thanks! I've seen those and am planning to restructure the whole thing in a unified way at some point. This was more of a quick fix for a downstream project.